plugins/generator-1.15.2/forge-1.15.2/procedures/entity_is_elytra_flying.java.ftl

```diff
@@ -0,0 +1 @@
+${input$entity}.isElytraFlying()
```

Comment:
As I wrote, this procedure is implemented in livingentity, base entity does not have this method as tests also told you. Please cast to living entity with appropriate checks same for the other procedure for both mc versions

Reply:
Yes ik just please stop marking my PR as draft the second after you spot an issue with it. Now that the issue is fixed, I'll un-draft it...

Reply:
Well, I use the draft tag to see what issues I need to check when I open PR page. Check and thoughtfully test the code and we won't need that many draft switches ;)